frontend/public/components/modals/remove-volume-modal.tsx

```diff
@@ -0,0 +1,81 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory';
+import {history, KebabAction, resourcePathFromModel} from '../utils';
+import {getVolumeType, K8sKind, K8sResourceKind, k8sUpdate} from '../../module/k8s/';
+import { VolumeData } from '../volumes-table';
+
+export class RemoveVolumeModal extends React.Component<RemoveVolumeModalProps, RemoveVolumeModalState> {
```

Comment:
Prefer functional components and hooks for new components. See #1456

Reply:
Now using FC and hooks for RemoveVolumeModal